### PR TITLE
Added Iterated Functions System (IFS) example

### PR DESCRIPTION
**Pull request by regislebrun**
This example illustrates the implementation of the optimal sampling stochastic algorithm to generate IFS fractals.

### CODE REVIEW

**Comment by jschueller** [comment]
Nice, maybe it should go in /numerical_methods

**Comment by jschueller** [comment]
Yes, just rebase and re-render the cells, as the default is now PNG.

**Comment by regislebrun** [comment]
done!

File: python/doc/examples/examples.rst [hunk 1]
```diff
@@ -31,4 +31,3 @@ This is the examples section.
         flood_model.ipynb
         optimal_lhs.ipynb
         ego.ipynb
-        ifs.ipynb
```
Comment (jschueller) [inline python/doc/examples/examples.rst]: You did not remove examples/ifs.ipynb

**Comment by dbarbier** [comment]
Please squash your commits together, otherwise original 10MB notebook is still in history and waste space.
When all changes are committed, run
```
  git rebase -i master ifs_example
```
This opens up an editor window with lines
```
pick 3ed53da8a Added Iterated Functions System (IFS) example
pick 8c1061c56 Moved ifs.ipynb to numerical_methods
```
(and maybe more commits) keep the first one and replace 'pick' by 'squash' in subsequent lines:
```
pick 3ed53da8a Added Iterated Functions System (IFS) example
squash 8c1061c56 Moved ifs.ipynb to numerical_methods
```
Then save and exit, this rewrites history, check result commit, and git push -f your branch.

**Comment by regislebrun** [comment]
@dbarbier Thanks for the instructions. I still have troubles with git when moving or deleting a file... Sorry.

**Pull request merged**